src/components/Bookings.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [submit, setSubmit] = useState("");
+
+  useEffect(() => {
```

Comment:
Similar to my comments in the `SearchResults` component. You can use state in this component to render appropriate `loading` and `error` messages on your website depending on the state of your API call.